Libplanet.Tests/Blocks/BlockTest.cs

```diff
@@ -133,57 +133,60 @@ public void ToBencodex()
         [Fact]
         public void FromBencodex()
         {
-            byte[] encoded = new byte[659]
+            byte[] encoded = new byte[697]
```

Comment:
This can be shorter:

```suggestion
            byte[] encoded =
```